src/libs/actions/SignInRedirect.js

```diff
@@ -44,6 +45,7 @@ function clearStorageAndRedirect(errorMessage) {
  * @param {String} [errorMessage] error message to be displayed on the sign in page
  */
 function redirectToSignIn(errorMessage) {
+    Network.clearRequestQueue();
```

Comment:
Please add a comment here to explain why this is here.

Reply:
Added the information in the function description